.github/workflows/charts.yml

```diff
@@ -0,0 +1,64 @@
+name: Charts
+
+on:
+  pull_request:
+    branches: [ main, release/** ]
+  push:
+    branches: [ main, release/** ]
+    tags: [ v* ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install ct
+        uses: helm/chart-testing-action@v2.3.1
+
+      - name: Run lint
+        run: ct lint --config .github/ct.yaml --charts=helm-chart
+
+  install:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install k3d
+        run: curl --retry 3 -fsL https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
+
+      - name: Create k3d cluster
+        run: k3d cluster create mirror --agents 1 --timeout 5m --registry-create registry:0.0.0.0:5001 --image rancher/k3s:v1.24.7-k3s1
+        timeout-minutes: 3
+
+      - name: Set up Docker Qemu
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
+
+      - name: Build and push images
+        uses: docker/build-push-action@v4
+        with:
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          platforms: linux/amd64, linux/arm64
+          push: true
+          tags: localhost:5000/${{ github.repository }}:test
+
+      - name: Install ct
+        uses: helm/chart-testing-action@v2.3.1
+
+      - name: Install chart
+        run: ct install --charts=helm-chart --helm-extra-args="--timeout 10m" --helm-extra-set-args="--set=image.registry=registry:5000 --set=image.tag=test --set=config.hosted.HEDERA_NEWORK={"127.0.01:50211":"0.0.3"} --set=config.MIRROR_NODE_URL=http://127.0.0.1:5551 --set=config.CHAIN_ID=0x12a"
```

Comment:
Maybe move set value to default if it's not possible to  provide a file for github action runs

Reply:
Added